rfcs/0000-User&HostIntegrationRFC.md

```diff
@@ -0,0 +1,254 @@
+asset.costCenter	| keyword |	Security - Protections | - | -                              | Cost Center associated with the asset (user or host).			
+asset.cost_center_hierarchy	| keyword |	Engineering	 | - | -                                | Additional cost center information associated with the asset (user or host).			
+asset.status	    | keyword         |	ACTIVE      | - | -                                 | Current status of the asset in the inventory datasource.			
+asset.last_status_change_date	| date |	June 5, 2023 @ 18:25:57.000	| - | -             | The most recent date/time when the asset.status was updated.			
+asset.create_date	            | date |	June 5, 2023 @ 18:25:57.001	| - | -             | For users, it's the hire date, for other assets, it's the in service date.			
+asset.end_date	                | date |	June 5, 2023 @ 18:25:57.002	| - | -             | For users, it's the termination date, for other assets, it's the out-of-service date.			
+asset.first_seen	            | date |	June 5, 2023 @ 18:25:57.003	| - | -             | The earliest date/time at which this asset was observed.  For assets added via config, this would remain empty until the asset was observed.			
+asset.last_seen	                | date |	June 5, 2023 @ 18:25:57.004	| - | -             | The most recent date/time at which this asset was observed.  Would remain empty until the asset was observed.			
+asset.last_updated	            | date |	June 5, 2023 @ 18:25:57.005	| - | -             | The most recent date/time this asset was updated in the inventory datasource.		
+asset.serial_number	            | keyword	| C02FG1G1MD6T	| - | -             |		Serial number of the asset.
+asset.tags	                    | keyword	  | watch, mdmaccess		| - | -             |	Tags assigned at the MDM.
+asset.assigned_users	          | keyword	  | user1@email.com, user2@email.com		| - | -             |	List of users assigned to the asset.
+asset.assigned_users_are_admin	| boolean	  | TRUE	| - | -             |		Flag to identify if the assigned users have admin priviledges.
+asset.is_managed	              | boolean	  | TRUE			| - | -             | If asset is managed by the organization.
+asset.last_enrolled_date	      | date	    | June 5, 2023 @ 18:25:57.005		| - | -             |	The most recent date/time the asset checked in with MDM.
+asset.data_classification	      | keyword	  | restricted		| - | -             |	Data classification tier for the asset.
+
+#### Nesting of existing risk.* fields under asset object
+* We have a set of risk.* fields in ECS which can be further nested under the asset.* object. Reference to [Risk RFC](https://github.com/elastic/ecs/blob/main/rfcs/text/0031-risk-fields.md).
+
+
+
+### Proposed New Fields for Host object
+Field | Type | Example | Description
+--- | --- | --- | ---
+host.os.build	| keyword		| 22F66   | Host OS Build information
```

Comment:
The current convention for build info is populating into the `host.os.kernel` (e.g. `winlogbeat` populates with `10.0.17763.4377 (WinBuild.160101.0800)` on a Windows Server 2019 Datacenter system). Is the idea here to create another field solely for the build number?

For this change as proposed, the field would go in as `os.build` under `os.*` and populate anywhere `os.*` fields are nested: `host.os.build`, `observer.os.build`, and `user_agent.os.build`.

Reply:
> Is the idea here to create another field solely for the build number?

Let's add this under `Concerns`. Adding and populating an `os.build` field isn't breaking, but it is a notably change from the current convention.

Reply:
Addressed this as part of this commit - https://github.com/elastic/ecs/pull/2215/commits/fd289d1eac3c22a64e80d18989d64443ca8a6814